$NetBSD$

Include <ncurses/term.h>, not <ncursesw/term.h>.
Check for HAVE_NCURSES_TERM_H, not for HAVE_NCURSES_CURSES_H.

Workaround for build failure on early NetBSD 6.99.* and prerelease 6.0
(PR 46492)

--- src/gui/curses/gui-curses-term.c.orig	2012-03-10 07:56:43.000000000 +0000
+++ src/gui/curses/gui-curses-term.c
@@ -25,9 +25,13 @@
 #include "config.h"
 #endif
 
-#ifdef HAVE_NCURSESW_CURSES_H
-#include <ncursesw/term.h>
+#ifdef HAVE_NCURSES_TERM_H
+#include <ncurses/term.h>
 #else
+#ifdef __NetBSD__
+/* workaround for build failure on early 6.99.* and prerelease 6.0 */
+#include <sys/cdefs.h>
+#endif
 #include <term.h>
 #endif
 
